qcsrc/doors.qc

```diff
@@ -62,7 +63,7 @@ void() door_blocked =
 
 void() door_hit_top =
 {
-	sound (self, CHAN_VOICE, self.noise1, 1, ATTN_NORM);
+	sound (self, CHAN_NO_PHS_ADD+CHAN_VOICE, self.noise1, 1, ATTN_NORM);
```

Comment:
What? Does QW see this as a valid channel?

Reply:
Yes. This is straight from the QuakeWorld codebase.

It's kind of weird. It's not actually a channel, it's a flag that modifies how the sound works. From what I understand, this makes it so that the sound packet will be sent to everyone in the server, not just to people within the potentially-hearable-set. (PHS)

Reply:
Thanks for clearing that up, marking as resolved because one of your previous comments describes a path to document this